src/quo/components/list_items/status_list_item/schema.cljs

```diff
@@ -0,0 +1,12 @@
+(ns quo.components.list-items.status-list-item.schema)
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:status [:enum :positive :negative]]
+      [:title [:maybe string?]]
+      [:description {:optional true} [:maybe string?]]
+      [:blur? {:optional true} [:maybe boolean?]]]]]
```

Comment:
As per [Figma](https://www.figma.com/design/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=16927-188939&m=dev), we also have a 'blur?' variant. However, as I mentioned in [Discord](https://discord.com/channels/1210237582470807632/1274068685266489434/1357663468559270081), the 'blur?' variant is the same as the 'dark' variant. I’ve still added it here so we can use the 'blur?' variant as per Figma in screens where it's used, and quickly update with new changes if needed.